README.md

```diff
@@ -6,6 +6,7 @@
   * [OrionHDFSSink persistence](#section3.1)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/078c59b46bfb6c3f30f6c21e98f51b287826071e